bundled-guava/NOTICE

```diff
@@ -5,8 +5,3 @@ Copyright 2017-2020 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
---------------------------------------------------------------------------------
```

Comment:
Is there a reason we don't mention that it contains Guava in this manner? Just curious.